index.html

```diff
@@ -4,18 +4,169 @@
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+        <div class="article-div">
+            <div class="text-div">
+                <h3>Daniel Matheus <i>wrote</i></h3>
+
+                <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+
+        <div class="membership-cont">
+            <div class="membership-ad">
+                <p>All you want is a <b>fast, secure lorem ipsum</b>? Dolor sit amet, consectetur adipiscing elit. In
+                    faucibus sit amet tellus nec efficitur.
+                    Maecenas consequat et libero at interdum. Ut pellentesque eget mi a finibus. Phasellus ac tincidunt
+                    ante. Suspendisse potenti. Mauris in tellus magna. Morbi quis magna elit. </p>
+                <ul>
+                    <li>Lorem ipsum dolor sit amet, consectetur <b>adipiscing elit.</b></li>
+                    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+                    <li>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</li>
+                </ul>
+            </div>
+            <div class="membership-img">
+
+            </div>
+        </div>
+
+        <div>
+            <div class="article-wrap">
+                <div class="article-div">
+                    <div class="text-div">
+                        <h1>Lorem ipsum dolor sit amet</h1>
+
+                        <h3>By: Daniel Matheus</h3>
+
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In faucibus sit amet tellus nec
+                            efficitur. Maecenas consequat et libero at interdum. Ut pellentesque eget mi a finibus.
+                            Phasellus ac tincidunt ante. Suspendisse potenti. Mauris in tellus magna. Morbi quis magna
+                            elit. Aliquam augue nulla, laoreet sit amet semper vitae, tincidunt dignissim nisl. </p>
+                    </div>
+                </div>
+                <div class="article-div">
+                    <div class="text-div">
+                        <h1>Lorem ipsum dolor sit amet</h1>
+
+                        <h3>By: Daniel Matheus</h3>
+
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In faucibus sit amet tellus nec
+                            efficitur. Maecenas consequat et libero at interdum. Ut pellentesque eget mi a finibus.
+                            Phasellus ac tincidunt ante. Suspendisse potenti. Mauris in tellus magna. Morbi quis magna
+                            elit. Aliquam augue nulla, laoreet sit amet semper vitae, tincidunt dignissim nisl. </p>
+                    </div>
+                </div>
+                <div class="article-div">
```

Comment:
This one also should be a `<section>` tag with its correspondent `<article>` tags, these apart from the requirements stated before should have these characteristics:

* [ ] Three articles with a width of `33%`